docker/Dockerfile.sycl

```diff
@@ -118,8 +118,8 @@ RUN SCRATCH_DIR=/scratch && mkdir -p ${SCRATCH_DIR} && cd ${SCRATCH_DIR} && \
     rm -rf ${SCRATCH_DIR}
 
 # Install Kokkos
-# FIXME hash below correspond to 3.5.99 (pre 3.6 release develop branch)
-ARG KOKKOS_VERSION=bb340781ba774eb0829de03735fe25e89b1176fc
+# FIXME hash below correspond to 3.6.99 (pre 3.7 release develop branch)
```

Comment:
That's past the release branch, we just haven't updated the version yet, see https://github.com/kokkos/kokkos/pull/5357.

Reply:
Which hash would you suggest?

Reply:
The hash is fine with me. I was just saying that the comment is imprecise.
```suggestion
# FIXME hash below corresponds to 3.6.99 (past 3.7 release develop branch)
```
or we wait for that pull request to be merged and say
```suggestion
# FIXME hash below corresponds to 3.7.99 (past 3.7 release develop branch)
```
or you pick a commit on the release branch (possibly waiting for https://github.com/kokkos/kokkos/pull/5358).